pkg/relayer/proxy/errors.go

```diff
@@ -3,9 +3,11 @@ package proxy
 import sdkerrors "cosmossdk.io/errors"
 
 var (
-	codespace                       = "relayer/proxy"
-	ErrUnsupportedRPCType           = sdkerrors.Register(codespace, 1, "unsupported rpc type")
-	ErrInvalidRelayRequestSignature = sdkerrors.Register(codespace, 2, "invalid relay request signature")
-	ErrInvalidSession               = sdkerrors.Register(codespace, 3, "invalid session")
-	ErrInvalidSupplier              = sdkerrors.Register(codespace, 4, "invalid supplier")
+	codespace                            = "relayer/proxy"
```

Comment:
@bryanchriswhite and I discussed this. Can you rename the errors to `ErrRelayerProxyXXX` so it's more semantic when reading the code, and the errors to just be a bit more descriptive: e.g. `invalid session in relay request`